contracts/samples/TokenPaymaster.sol

```diff
@@ -124,7 +124,8 @@ contract TokenPaymaster is BasePaymaster, UniswapHelper, OracleHelper {
             require(dataLength == 0 || dataLength == 32,
                 "TPM: invalid data length"
             );
-            uint256 preChargeNative = requiredPreFund + (tokenPaymasterConfig.refundPostopCost * userOp.maxFeePerGas);
+            (uint256 maxFeePerGas,) = UserOperationLib.unpackAccountGasLimits(userOp.gasFees);
```

Comment:
I think it is a problem using a function named `unpackAccountGasLimits` for unpacking fees per gas - please rename it to something more generic, maybe?

Reply:
thanks.
any suggested naming?
probably: `unpackPrioAndFeePerGas() returns (maxPrio, maxFeePerGas)`

Reply:
Are you adding a function or renaming existing one? If you add, maybe unpackGasFees?